pkg/controller/compute/scheduler/scheduler_test.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2018 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License
+*/
+
+package scheduler
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/crossplaneio/crossplane/pkg/apis/compute"
+	. "github.com/crossplaneio/crossplane/pkg/apis/compute/v1alpha1"
+	corev1alpha1 "github.com/crossplaneio/crossplane/pkg/apis/core/v1alpha1"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	. "sigs.k8s.io/controller-runtime/pkg/client/fake"
```

Comment:
Do we really want three anonymous imports in the same file? It makes it difficult to ascertain where functions come from to the untrained eye. At first I struggled to determine where `NewFakeClient()` came from before I noticed we were anonymously importing more than just `gomega`.

(Cards on the table: I don't ever like anonymous imports. It sucks to type package names over and over, but it's worth it in my opinion for the increased readability.)

Reply:
I share the same sentiment about anonymous imports. With that said, if we have package collision (such as `fake` or `errors` or `testing`) we forced to use package aliases. I am not defending `.` import per say, but I am not sure which is worse `.` vs . `fake2`